### PR TITLE
deploy  with capistrano

### PR DESCRIPTION
**Pull request by talits**
Deploy with capistrano

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 /app/config/parameters.yml
 /build/
 /phpunit.xml
+/bin/*
```
Comment (marcusesa) [inline .gitignore]: We have some important files in bin directory such as console file, maybe is better to be specific. What files do you want ignore?

File: Capfile [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# For documentation on these, see for example:
+#
+#   https://github.com/capistrano/rvm
+#   https://github.com/capistrano/rbenv
+#   https://github.com/capistrano/chruby
+#   https://github.com/capistrano/bundler
+#   https://github.com/capistrano/rails
+#   https://github.com/capistrano/passenger
+#
+# require "capistrano/rvm"
```
Comment (marcusesa) [inline Capfile]: If the comment isn't necessary, is better to remove them.

File: composer.json [hunk 3]
```diff
@@ -41,8 +41,6 @@
         "symfony-scripts": [
             "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
```
Comment (marcusesa) [inline composer.json:44]: Why did you remove those lines? Did you have some problems with them?

File: composer.json [hunk 4]
```diff
@@ -59,7 +57,7 @@
     "extra": {
         "symfony-app-dir": "app",
         "symfony-bin-dir": "bin",
-        "symfony-var-dir": "var",
+        "symfony-var-dir": "var" ,
```
Comment (marcusesa) [inline composer.json]: White space unnecessary.

File: config/deploy.rb [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# config valid only for current version of Capistrano
+lock "3.8.2"
+
+# config valid only for current version of Capistrano
+set :stages, ["prod","prod-php5"]
+set :symfony_env,  "prod"
+set :branch, "master"
+set :application, ask("set the correct app:")
+set :repo_url, "git@github.com:Talits/qedu-hub.git"
```
Comment (marcusesa) [inline config/deploy.rb]: We need update `repo_url` value to `git@github.com:QEdu/qedu-hub.git`

File: config/deploy.rb [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# config valid only for current version of Capistrano
+lock "3.8.2"
```
Comment (marcusesa) [inline config/deploy.rb:2]: Nice to use the latest version.

File: config/deploy.rb [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+# config valid only for current version of Capistrano
+set :stages, ["prod","prod-php5"]
```
Comment (marcusesa) [inline config/deploy.rb:5]: What does **prod-php5** mean?

File: config/deploy.rb [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+set :symfony_console_path, "bin/console"
+set :permission_method, :acl
+set :permission_method, true
+#set :file_permissions_paths, ["app/logs", "app/cache"]
+set :file_permissions_paths, [fetch(:log_path), fetch(:cache_path), "var", "web/uploads"]
+set :file_permissions_users, ["nginx"]
+
+
+after "deploy:log_revision", "dep:node"
+after "dep:node", "shared:googlewmt"
```
Comment (marcusesa) [inline config/deploy.rb]: What does **googlewmt** mean?

Comment (talits) [reply config/deploy.rb]: It's a test, sorry

File: .gitignore (hunk 1 above)
Comment (talits) [reply .gitignore]: perfect

File: Capfile (hunk 2 above)
Comment (talits) [reply Capfile]: ok

File: composer.json (hunk 3 above)
Comment (talits) [reply composer.json:44]: I have changed before, but I don't have a problem

File: composer.json (hunk 4 above)
Comment (talits) [reply composer.json]: ok

File: config/deploy.rb (hunk 5 above)
Comment (talits) [reply config/deploy.rb]: perfect

File: config/deploy.rb (hunk 7 above)
Comment (talits) [reply config/deploy.rb:5]: It's a role for the app legacy

**Comment by marcusesa** [comment]
@Talits 85 files have changed, I think you changed file permissions unintentionally.
